modules/navigation/SCsub

```diff
@@ -36,12 +36,7 @@ if env["builtin_recastnavigation"]:
 # RVO 2D Thirdparty source files
 if env["builtin_rvo2_2d"]:
     thirdparty_dir = "#thirdparty/rvo2/rvo2_2d/"
-    thirdparty_sources = [
-        "Agent2d.cpp",
-        "Obstacle2d.cpp",
-        "KdTree2d.cpp",
-        "RVOSimulator2d.cpp",
-    ]
+    thirdparty_sources = ["Agent2d.cc", "Obstacle2d.cc", "KdTree2d.cc", "Line.cc", "RVOSimulator2d.cc", "Vector2.cc"]
```

Comment:
Please keep those with one per line. You need to use a trailing comma for the last item.